views/views.go

```diff
@@ -49,36 +52,83 @@ type (
 
 	// Repo defines all view interactions
 	Repo interface {
```

Comment:
Similar comment as earlier - what's the purpose of this interface? (I appreciate it's not your code so may be better for a follow-up rather than this PR, but there's no reason to have an interface with only one implementor except in a library.)

Reply:
I'll just remove them as they are easy to make again